api/src/main/scala/hmda/api/http/institutions/submissions/SubmissionSignPaths.scala

```diff
@@ -85,8 +93,58 @@ trait SubmissionSignPaths
 
     onComplete(fSubmission) {
       case Success(sub) =>
+        if (sub.status.code == 10) {
+          emailSignature(supervisor, sub)
+        }
         complete(ToResponseMarshallable(Receipt(sub.end, sub.receipt, sub.status)))
       case Failure(error) => completeWithInternalError(uri, error)
     }
   }
+
+  private def emailSignature(supervisor: ActorRef, submission: Submission)(implicit ec: ExecutionContext) = {
+    val emails = findEmailsById(submission.id.institutionId)
+    emails.map(emailSeq => {
+      emailSeq.foreach(t => {
+        val username = t._1 + " " + t._2
+        sendMail(t._3, username, submission)
+      })
+    })
+  }
+
+  private def sendMail(address: String, username: String, submission: Submission) = {
```

Comment:
The team implied that the platform will eventually have other reasons to send email. Could this functionality move into a module so we can reuse it?

Reply:
I'm going to go with the same approach we used for the A&D reports.  I'll leave it here until now, and if we later need it in another section of the code, we can work on it then.

Reply:
Cool, in that case, please add a GitHub issue for that work.